qutebrowser/browser/browsertab.py

```diff
@@ -560,6 +560,7 @@ def __init__(self, win_id, mode_manager, parent=None):
         self._mouse_event_filter = mouse.MouseEventFilter(
             self, widget_class=self.WIDGET_CLASS, parent=self)
         self.backend = None
+        self.pin = False
```

Comment:
I'd prefer this to be called `pinned` - also it should probably be in the `TabData` class instead, which is accessible as the `.data` attribute of the `BrowserTab` object - otherwise, `BrowserTab` would have a lot of quite "random" attributes :wink: